src/topology/path_connected.lean

```diff
@@ -372,6 +402,24 @@ begin
     simp [hst, mul_inv_cancel (@two_ne_zero ℝ _ _)] }
 end
 
+@[continuity]
+lemma continuous_trans {x y z : X} : continuous (λ ρ : path x y × path y z, ρ.1.trans ρ.2) :=
+continuous_uncurry_iff.mp begin
+  apply trans_continuous_family;
+  refine (continuous.comp _ continuous_fst).path_eval continuous_snd,
+  exacts [continuous_fst, continuous_snd],
+end
+
+@[continuity]
+lemma _root_.continuous.path_trans (Y₁ Y₂ : Type) [topological_space Y] (f : Y → path x y)
+  (g : Y → path y z) : continuous f → continuous g → continuous (λ t, (f t).trans (g t)) :=
```

Comment:
```suggestion
lemma _root_.continuous.path_trans [topological_space Y] (f : Y → path x y) (g : Y → path y z) : 
  continuous f → continuous g → continuous (λ t, (f t).trans (g t)) :=
```

Reply:
With this continuity lemma, I think `continuous_trans` then needn't be a continuity lemma? I'm not sure how usually things are done.

Reply:
I leave this to @urkud to decide.